hooks/command

```diff
@@ -34,8 +34,11 @@ if [[ -z ${files:-} ]] ; then
   exit 1
 fi
 
+# Read in the options to pass to shellcheck
+mapfile -t options<<<"$(plugin_read_list OPTIONS)"
```

Comment:
I'm mildly concerned that this pattern comes with a bash 4.x+ requirement.

Reply:
If I'm not mistaken, does this mean I can't run the shellcheck plugin anymore via `bksr` on my Mac? (`bash -v`)

Reply:
Sorry, I didn't realise mapfile was 4.x+. This has been merged already but if this is an issue you should be able to change it back to `options=( $(plugin_read_list OPTIONS) )`, which ironically wouldn't pass SC2207.

Reply:
Fixed in #15.

Reply:
Also updated the Shellcheck wiki 😅https://github.com/koalaman/shellcheck/wiki/SC2207

Reply:
🙏🏼